toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1812,6 +1812,8 @@ func SELinuxUpdateConfig(selinuxMode configuration.SELinux, installChroot *safec
 		mode = SELinuxConfigEnforcing
 	case configuration.SELinuxPermissive:
 		mode = SELinuxConfigPermissive
+	case configuration.SELinuxOff:
+		mode = SELinuxConfigDisabled
```

Comment:
Since we're on `SELinuxConfigDisabled`, could you also update the comments above this and the `SELinuxConfigPermissive` constants in `imagegen/installutils/installutils.go`? They both start with "// **SELinuxConfigEnforcing** is the string..." - it's just about using the correct constant name.

Reply:
Fixed.